docs/src/main/sphinx/release/release-476.md

```diff
@@ -0,0 +1,171 @@
+* Improve compatibility with S3-compliant storage systems ({issue}`25791`)
+* Show detailed metrics from splits generation in output of EXPLAIN ANALYZE VERBOSE. ({issue}`25770`)
+* Add `max_partitions_per_writer` session property, which corresponds to the `iceberg.max-partitions-per-writer` configuration property. ({issue}`25662`)
+* Improve query planning performance when reading from materialized views. ({issue}`24734`)
+* Fix rare failure when `iceberg.bucket-execution` is enabled. ({issue}`25125`)
+* Fix query failure when reading ORC files with a large row count. ({issue}`25634`)
+* Reduce S3 throttling failures by increasing streaming upload part size and reducing number of requests. ({issue}`25781`)
+* Fix query timeout errors due to concurrent writes on tables with large number of manifest files. ({issue}`24751`)
+* Fix query failures when attempting to read parquet data with integer logical annotation as a `date` column. ({issue}`25667`)
+
+## Ignite connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## JMX connector
+
+## Kafka connector
+
+## Kafka Event Listener
+
+* `kafka-event-listener.client-config-overrides` is removed. To configure the Kafka client for the event listener specify the configuration in a separate file and set `kafka-event-listener.config.resources` to the path to the file. ({issue}`25553`)
+
+## Loki connector
+
+## MariaDB connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Memory connector
+
+* Fix failures when deleting rows from memory connector ({issue}`25670`)
+
+## MongoDB connector
+
+## MySQL connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## OpenSearch connector
+
+## Oracle connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+* Improve performance of listing table columns. ({issue}`25231`)
+
+## Pinot connector
+
+## PostgreSQL connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Prometheus connector
+
+## Redis connector
+
+## Redshift connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## SingleStore connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Snowflake connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+* {{breaking}} Require `--sun-misc-unsafe-memory-access=allow` while running Trino server. ({issue}`25669`)
+
+## SQL Server connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## TPC-H connector
+
+## TPC-DS connector
+
+## Vertica connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## SPI
+
+* Block `getSizeInBytes` now returns an estimate of the full data size of the block. ({issue}`25256`)
```

Comment:
Something like

Return an estimate of the full data size of the block with `getSizeInBytes`.